packages/e2e/cypress/e2e/app/embed.cy.ts

```diff
@@ -0,0 +1,79 @@
+import {
+    CreateEmbedJwt,
+    FilterInteractivityValues,
+    SEED_PROJECT,
+} from '@lightdash/common';
+
+const getEmbedUrl = (body: CreateEmbedJwt) =>
+    cy.request({
+        url: `/api/v1/embed/${SEED_PROJECT.project_uuid}/get-embed-url`,
+        headers: { 'Content-type': 'application/json' },
+        method: 'POST',
+        body,
+    });
+
+const getJaffleDashboard = () =>
+    cy.request(
+        `/api/v2/content?pageSize=1&contentTypes=dashboard&search=jaffle`,
+    );
+
+describe('Embedded dashboard', () => {
+    before(() => {
+        cy.login();
+    });
+    it('I can view embedded dashboard and all interactivity options', () => {
+        getJaffleDashboard().then((dashboardsResp) => {
+            const dashboardUuid = dashboardsResp.body.results.data[0]?.uuid;
+            getEmbedUrl({
+                content: {
+                    type: 'dashboard',
+                    dashboardUuid,
```

Comment:
don't we need to make sure first this dashboard is int he list of embedded dashboards ? This works because you are filtering by `jaffle` , but that makes it a bit unstable if we start adding dashboards or doing manual testing.

Reply:
I guess it works for now, but if we startt having issues, we can change it later.

Reply:
Leaving it for now. It is the same assumption as most e2e tests that rely on this dashboard. Any change to the dashboard/chart will cause e2e tests to fail.